hal/src/main/native/athena/AnalogGyro.cpp

```diff
@@ -14,6 +14,7 @@
 #include "hal/AnalogAccumulator.h"
 #include "hal/AnalogInput.h"
 #include "hal/handles/IndexedHandleResource.h"
+#include <wpi/raw_ostream.h>
```

Comment:
Header is in the wrong location. Headers are sorted by related headers, C system, C++ system, other libs, project headers in that order where each group is separated by newlines. raw_ostream.h is considered from "other libs". I recommend checking the styleguide job immediately instead of waiting for the whole thing to finish since the styleguide errors are found right away.